DEVELOPMENT.md

```diff
@@ -0,0 +1,39 @@
+# Development Tips
+
+## Code Generation
+For matching interface between SDK and Henesis API server, SDK uses `codegen`. So, if you add/update Henesis API server, run command to generate interface codes automatically. 
+```bash
+# in pakcages/core
+npm run codegen
+```
+The interface codes are generated in `packages/core/src/__generate__`.
```

Comment:
"code" is uncountable.
How about using these phrases "the interfaces below are" or "The interface code is" instead of "The interface codes are"

Reply:
This [link](https://english.stackexchange.com/questions/20455/is-it-wrong-to-use-the-word-codes-in-a-programming-context/) may help you.

Reply:
wow